### PR TITLE
Cockpit 151

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-151.html

### CODE REVIEW

File: _posts/2017-09-21-cockpit-151.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Support loading SSH keys from arbitrary paths
+
+The User menu's Authentication dialog now supports entering arbitrary paths to
+SSH keys for adding to the SSH authentication agent. Previously this only
+offered keys present in the standard `~/.ssh` home directory.
+
+See it in action:
+
+<iframe width="960" height="720" src="https://cockpit.fedorapeople.org/add-keys.webm" frameborder="0" allowfullscreen></iframe>
```
Comment (stefwalter) [inline _posts/2017-09-21-cockpit-151.md]: Is this hosting location long lasting enough? Any reason not to use Youtube? I realize @petervo recorded it like this. Worth uploading elsewhere?

File: _posts/2017-09-21-cockpit-151.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### Support X-Forwarded-Proto HTTP header for Kubernetes
+
+Newer Kubernetes versions support reading the
+[X-Forwarded-Proto](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto)
+HTTP header, which helps to determine whether or not a client used SSL to
+connect to an intermediate proxy, load balancer, or similar. Cockpit's
+Kubernetes (Cluster) dashboard now sets this header. Earlier versions have
+already done that for connecting to OpenShift.
```
Comment (stefwalter) [inline _posts/2017-09-21-cockpit-151.md]: "already done that when hosted in Openshift"

Comment (martinpitt) [reply _posts/2017-09-21-cockpit-151.md]: Updated, thanks!

File: _posts/2017-09-21-cockpit-151.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2017-09-21-cockpit-151.md]: It feels a bit presumptuous to me to upload it to youtube in his stead. I don't see why we couldn't leave it there, or we could update the blog post if/when @petervo uploads it to YouTube. I don't think we need to block the release blog for this.

**Pull request merged**